### PR TITLE
update with zipped indices

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
    - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
    - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
    - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by SusiJo — APPROVED** [review]
Nice! 🚀

File: conf/igenomes.config [hunk 1]
```diff
@@ -53,8 +53,8 @@ params {
             dict                  = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Sequence/WholeGenomeFasta/Homo_sapiens_assembly38.dict"
             fasta                 = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Sequence/WholeGenomeFasta/Homo_sapiens_assembly38.fasta"
             fasta_fai             = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Sequence/WholeGenomeFasta/Homo_sapiens_assembly38.fasta.fai"
-            germline_resource     = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/GermlineResource/gnomAD.r2.1.1.GRCh38.PASS.AC.AF.only.vcf.gz"
-            germline_resource_tbi = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/GermlineResource/gnomAD.r2.1.1.GRCh38.PASS.AC.AF.only.vcf.gz.tbi"
+            germline_resource     = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/GATKBundle/af-only-gnomad.hg38.vcf.gz"
+            germline_resource_tbi = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh38/Annotation/GATKBundle/af-only-gnomad.hg38.vcf.gz"
```
Comment (asp8200) [inline conf/igenomes.config:57]: Should that have been a tbi-file?

Comment (FriederikeHanssen) [reply conf/igenomes.config:57]: 🙈 yes, good catch. I'll update

**Comment by asp8200** [comment]
I just tried to see if I could download the newly referenced file `af-only-gnomad.hg38.vcf.gz`, but it didn't work out:

```
$ aws s3 --no-sign-request cp s3://ngi-igenomes/igenomes/Homo_sapiens/GATK/GRCh38/Annotation/GATKBundle/af-only-gnomad.hg38.vcf.gz .
fatal error: An error occurred (404) when calling the HeadObject operation: Key "igenomes/Homo_sapiens/GATK/GRCh38/Annotation/GATKBundle/af-only-gnomad.hg38.vcf.gz" does not exist
```

By doing the following command, I was able to download the GATKBundle, but it didn't include `af-only-gnomad.hg38.vcf.gz` :
```
aws s3 --no-sign-request cp s3://ngi-igenomes/igenomes/Homo_sapiens/GATK/GRCh38/Annotation/GATKBundle . --recursive
```
What am I doing wrong? Isn't the file `af-only-gnomad.hg38.vcf.gz` included in the `GATKBundle` yet?

**Comment by FriederikeHanssen** [comment]
the files didn't actually copy over apparently. I wish the aws cli would then actually fail and report that no files were moved.

**Review by maxulysse — APPROVED** [review]
OK to merge when files are actually there

**Review by maxulysse — APPROVED** [review]

**Review by apeltzer — APPROVED** [review]

**Comment by maxulysse** [comment]
Can we add the new `known_snps` to https://github.com/nf-core/sarek/blob/dev/docs/usage.md#where-do-the-used-reference-genomes-originate-from

**Review by maxulysse — APPROVED** [review]

**Pull request merged**